sysdeps/sigma/generic/memory.cpp

```diff
@@ -0,0 +1,25 @@
+#include <string.h>
+
+#include <bits/ensure.h>
+#include <mlibc/sysdeps.hpp>
+#include <mlibc/debug.hpp>
+#include <libsigma/memory.h>
+#include <errno.h>
+
+#define STUB_ONLY { __ensure(!"STUB_ONLY function was called"); __builtin_unreachable(); }
+
+namespace mlibc
+{
+    int sys_anon_allocate(size_t size, void **pointer){
+        __ensure((size & 0xFFF) == 0);
+        uint64_t ret = libsigma_valloc(LIBSIGMA_VALLOC_TYPE_SBRK_LIKE, 0, size / 0x1000);
+        if(ret == 1) return ENOMEM;
```

Comment:
This coding style is inconsistent with mlibc. Put the "then case" onto the next line.